query/src/lib.rs

```diff
@@ -68,6 +69,12 @@ impl Variable {
     }
 }
 
+impl fmt::Debug for Variable {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
I wonder if we could we solve that with

    write!(f, "{} #_ (variable)", self.0)

Or something similar?
(It feels like it's more natural that `PlainSymbol` should be the default.